packages/contracts/contracts/TellerV2.sol

```diff
@@ -148,7 +149,7 @@ contract TellerV2 is
 
     /** Constant Variables **/
 
-    uint256 public constant CURRENT_CODE_VERSION = 7;
+    uint8 public constant CURRENT_CODE_VERSION = 7;
```

Comment:
what is the point of this ?  this could cause storage slot shifting lets just leave it alone

Reply:
its a constant variable - no storage slots are used